packages/compiler/server/serverlib.ts

```diff
@@ -531,6 +535,45 @@ export function createServer(host: ServerHost): Server {
     }
   }
 
+  async function addImportCompletion(
+    program: Program,
+    document: TextDocument,
+    completions: CompletionList
+  ) {
+    const new_path = getPath(document);
+    const projectroot = await findProjectRoot(compilerHost, new_path);
```

Comment:
I would say to future proof this too should maybe move most of this into a new funciton `addLibraryImportCompletion` so we can have another `addRelativePathImportCompletion` and `addImportCompletion` would just call out to both